Tests/ApolloCodegenTests/ApolloCodegenTests.swift

```diff
@@ -211,6 +211,8 @@ class ApolloCodegenTests: XCTestCase {
   }
 
   func test_CCN_compileResults_givenOperations_withNoErrors_shouldReturn() throws {
+    throw XCTSkip("CCN tests skipped until issue #3114 done or closed.")
```

Comment:
These skipped tests will be handled in #3151 before `feature/defer` is merged into `main`.